orcid-persistence/src/main/resources/db/updates/org_disambiguated_extra_columns.xml

```diff
@@ -2,6 +2,14 @@
                    xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-2.0.xsd">
 
     <changeSet id="ADD-ORG-EXTRA-COLUMNS" author="Camelia Dumitru" dbms="postgresql">
+		 <preConditions onFail="MARK_RAN">
```

Comment:
@Camelia-Orcid just a comment here, this will change the hash of the liquibase file, so, liquibase will complain about it the next time it runs in QA/Sbox and PROD. Do we really need to add this at this point? It already ran in all environments.

Reply:
Removed, this change was needed in dev only and committed by mistake.